tools/runqslower.py

```diff
@@ -42,14 +42,17 @@
 examples = """examples:
     ./runqslower         # trace run queue latency higher than 10000 us (default)
     ./runqslower 1000    # trace run queue latency higher than 1000 us
-    ./runqslower -p 123  # trace pid 123 only
+    ./runqslower -p 123  # trace pid 123 and its children threads
+    ./runqslower -t 123  # trace tid 123 (use for threads only)
 """
 parser = argparse.ArgumentParser(
     description="Trace high run queue latency",
     formatter_class=argparse.RawDescriptionHelpFormatter,
     epilog=examples)
 parser.add_argument("-p", "--pid", type=int, metavar="PID", dest="pid",
-    help="trace this PID only")
+    help="trace this PID and its children")
+parser.add_argument("-t", "--tid", type=int, metavar="TID", dest="tid",
+    help="trace this TID only")
```

Comment:
"-p" and "-t" are exclusive. Please use `parser.add_mutually_exclusive_group`. See examples in offcputime.py.